weave-js/src/components/PagePanelComponents/Home/Browse2/RunsTable.tsx

```diff
@@ -490,14 +500,29 @@ export const RunsTable: FC<{
             return rowParams.row.call.spanName;
           }
           return (
-            <CallLink
-              entityName={params.entity}
-              projectName={params.project}
-              opName={opVersionRefOpName(opVersion)}
-              callId={rowParams.row.id}
-              fullWidth={true}
-              preservePath={preservePath}
-            />
+            <>
+              <Checkbox
```

Comment:
I am wondering if we always want to show this. Seems like a lot of attention to draw and a super valuable real estate to takeup for an action that is an edge case. Something we should discuss with design tomorrow maybe?